dev/system-test/firestore.ts

```diff
@@ -1355,6 +1356,19 @@ describe('Query class', () => {
     });
   });
 
+  it('stream() supports readable[Symbol.asyncIterator]()', async () => {
+    let received = 0;
+    await randomCol.doc().set({foo: 'bar'});
+    await randomCol.doc().set({foo: 'bar'});
+
+    const stream = randomCol.stream();
+    for await (const chunk of stream) {
```

Comment:
Does TypeScript compile it into something that older versions of Node.js can understand? Async iterators only appeared in Node 10, right?

Reply:
I am a little surprised too, but it seems to work. I stole this test from https://github.com/googleapis/nodejs-storage/pull/799

Reply:
Typescript transpiles it away:

```
  it('stream() supports readable[Symbol.asyncIterator]()', async () => {
        var e_1, _a;
        let received = 0;
        await randomCol.doc().set({ foo: 'bar' });
        await randomCol.doc().set({ foo: 'bar' });
        const stream = randomCol.stream();
        try {
            for (var stream_1 = __asyncValues(stream), stream_1_1; stream_1_1 = await stream_1.next(), !stream_1_1.done;) {
                const chunk = stream_1_1.value;
                ++received;
            }
        }
        catch (e_1_1) { e_1 = { error: e_1_1 }; }
        finally {
            try {
                if (stream_1_1 && !stream_1_1.done && (_a = stream_1.return)) await _a.call(stream_1);
            }
            finally { if (e_1) throw e_1.error; }
        }
        chai_1.expect(received).to.equal(2);
    });
```

Reply:
Also verifies that the test does indeed work with Symbol.asyncIterator by changing the target level to ES2019.